homeassistant/components/starlink/strings.json

```diff
@@ -0,0 +1,17 @@
+{
+  "config": {
+    "abort": {
+      "already_configured": "Starlink terminal is already configured"
+    },
+    "error": {
+      "server_error": "Could not reach Starlink terminal."
```

Comment:
Please use the standard `key::common` messages here

Reply:
Marking as resolved so we only have one thread for this